docs/i2c.rst

```diff
@@ -0,0 +1,29 @@
+I²C
+***
+
+.. py:module:: microbit.i2c
+
+The ``i2c`` module lets you communicate with devices connected to your board
+using the I²C bus protocol.There can be multiple devices connected at the same
+time, and each one has its own unique address, that is either fixed for the
+device or configured on it. How exactly you should communicate with those
+devices, that is, what bytes to send and how to interpret the responses,
+depends on the device in question and should be described separately in that
+device's documentation.
+
```

Comment:
Are there internal pull ups in the I2C lines? knowing if additional components are needed to connect to the I2C bus should be specified somewhere.